app/population/middleware/set-breadcrumb.js

```diff
@@ -30,25 +30,26 @@ function setBreadcrumb(req, res, next) {
     dateSuffix = ` (${formatDate(dateAsStartOfWeekDate)})`
   }
 
-  const basePath = `/population/week/${format(
+  const weeklyBasePath = `/population/week/${format(
     dateAsStartOfWeekDate,
     'yyyy-MM-dd'
   )}`
+  const dailyBasePath = `/population/${period}/${req.date}`
 
   res
     .breadcrumb({
       text: `${req.t('population::breadcrumbs.home')}${dateSuffix}`,
-      href: basePath,
+      href: weeklyBasePath,
     })
     .breadcrumb({
       text: locationName,
-      href: `${basePath}/${locationId}`,
+      href: `${weeklyBasePath}/${locationId}`,
     })
 
   if (period === 'day') {
     res.breadcrumb({
       text: format(parseISO(req.date), 'EEEE d MMMM'),
-      href: req.baseUrl,
+      href: `${dailyBasePath}/${locationId}`,
```

Comment:
Apologies for not following this more closely - does this mean there's no daily breadcrumb without a location? So there's no daily view of all locations?

Reply:
Yeah, this is for the PMU screens, so uses `/population/week/${date}` as an initial base - that will show all populations currently selected, in a table, including "All regions" if selected. A specific view for a location for a week showing transfers in/out as well is `/population/week/${date}/${locationId}`. Those are all collection based views. 

Drilling down further into a daily view for a location uses `/population/daily/${date}${locationId}`, which is a single item based view.

One of the weirdnesses with naming this section is that population is singular and plural. Other concepts such as "moves" have an item path of `/move` and a collection view of `/moves` which makes things easier to understand as they are more separated out.